Dockerfile

```diff
@@ -24,8 +24,32 @@ RUN rm -rf /etc/collectd
 # Setup our collectd
 ADD configs /etc/
 
+#Install unzip and pip
```

Comment:
As much as possible, chain consecutive RUN statements into a single command that uses `&&`, otherwise you get an explosion of Docker image layers